test.js

```diff
@@ -779,45 +781,89 @@ describe('Satellite()', () => {
       });
       helmetService.start(port, async () => {
         const res = await fetch(`${url}/always-200`);
+        // console.log(res);
         expect(res.ok).toBe(true);
         expect(res.headers.get('x-xss-protection')).toBe(null);
         helmetService.stop(done);
       });
     });
   });
-});
 
-describe('logger', () => {
-  test('logger should have expected methods()', () => {
-    ['trace', 'debug', 'info', 'warn', 'error', 'fatal'].forEach((level) => {
-      expect(typeof logger[level] === 'function').toBe(true);
-      expect(() => logger[level]('test')).not.toThrow();
+  describe('cache', () => {
+    test('forever() should set the max-age to 1y, and add the immutable tag', (done) => {
+      const service = createSatelliteInstance({
+        name: 'header-test',
+        port,
+      });
+      service.start(port, async () => {
+        const res = await fetch(`${url}/always-200`);
+        expect(res.ok).toBe(true);
+        cache().forever(res);
```

Comment:
You have to add the middleware to a new route.  Something like this:

```js
    const service = createSatelliteInstance({
      name: 'test',
    });
    const router = service.router;
    router.get('/cached-forever', cache.forever(), (req, res) => res.json({ hello: 'public' }));
```

Now you can try and `fetch()` that and check the headers:

```js
    service.start(port, async () => {
      let res = await fetch(`${url}/cached-forever`);
      expect(res.ok).toBe(true);
      expect(res.headers.get('Cache-Control')).toBe('public, max-age=31557600000, immutable');
      ...
```